content/applications/inventory_and_mrp/manufacturing/management/using_work_centers.rst

```diff
@@ -0,0 +1,126 @@
+- :guilabel:`Alternative Workcenters`: specify an alternative work center for operations to be
+  carried out at if the main work center is not available
+- :guilabel:`Code`: assign the work center a reference code
+- :guilabel:`Working Hours`: define the number of hours that the work center can be in use each week
+- :guilabel:`Company`: select the company that the work center belongs to
+
+.. image:: using_work_centers/work-center-form.png
+   :align: center
+   :alt: An example of a fully configured work center form.
+
+Set standards for work center productivity
+------------------------------------------
+The :guilabel:`General Information` tab on the work center form allows for productivity goals to be
+assigned to a work center:
+
+- :guilabel:`Time Efficiency`: used to calculate the expected duration of a work order at the work
+  center; for example, if a work order normally takes one hour and the efficiency is set to 200%,
+  the work order will take 30 minutes
+- :guilabel:`Capacity`: the number of operations that can be performed at the work center
+  simultaneously
+- :guilabel:`OEE Target`: the target for efficiency at the work center
+- :guilabel:`Time before prod.`: setup time required before work can commence
+- :guilabel:`Time after prod.`: breakdown or cleanup time required after work is finished
+- :guilabel:`Cost per hour`: the cost of operating the work center for one hour
+- :guilabel:`Analytic Account`: the account where the cost of the work center should be recorded
+
+.. image:: using_work_centers/work-center-general-information.png
+   :align: center
+   :alt: The general information tab of the work center form.
+
+Assign equipment to a work center
+---------------------------------
+Using the :guilabel:`Equipment` tab, it is possible for specific pieces of equipment to be assigned
+to a work center. The following information will be displayed for each piece of equipment added:
+
+- :guilabel:`Equipment Name`: the name of the piece of equipment
+- :guilabel:`Technician`: the technician responsible for servicing the equipment
+- :guilabel:`Equipment Category`: the category the equipment belongs to
+- :guilabel:`MTBF`: mean time between failures; the average time that the piece of equipment will
+  operate before failing
+- :guilabel:`MTTR`: mean time to recovery; the average time it takes for the equipment to become
+  fully operational again
+- :guilabel:`Est. Next Failure`: an estimate of when the next equipment failure will occur
+
+.. image:: using_work_centers/work-center-equipment.png
+   :align: center
+   :alt: The equipment tab of the work center form.
+
+.. note::
+    :guilabel:`MTBF`, :guilabel:`MTTR`, and :guilabel:`Est. Next Failure` are all calculated
+    automatically based on past failure data, if any exists.
+
+Integrate IoT devices
+---------------------
+The :guilabel:`IoT Triggers` tab enables the integration of :abbr:`IoT (Internet of Things)` devices
+with a work center:
```

Comment:
Too lean.

What is IoT? Why do we care? How does IoT integrate with Odoo work centers? 

Are there any best-practice IoT configurations we can mention here (can we hook up sensors to detect gas leaks, for example)? What's the example you used in the screenshot...maybe explain that one?